### PR TITLE
Added GET and POST endpoints for penalties API

### PR DESCRIPTION
**Pull request by ryanbarlow1**

### CODE REVIEW

File: backend/internal/server/apis/stats/penalty.go
Comment (Skylude) [inline backend/internal/server/apis/stats/penalty.go:1]: ⚠️ I would think we'd probably want this in a penalties folder vs a stats folder.

File: backend/internal/server/apis/stats/penalty.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	apis.RegisterHandler(fiber.MethodPost, "/penalties", auth.Public, postPenaltyHandler)
+}
+
+func getPenaltiesHandler(c *fiber.Ctx) error {
+	log := locals.Logger(c)
+	log.Info("Handling getting all penalties")
+	db := db.GetSession(c)
+	penalties, err := db.GetPenalties()
+	if err != nil {
+		log.WithErr(err).Alert("Failed to get all penalties from the database")
```
Comment (Skylude) [inline backend/internal/server/apis/stats/penalty.go:27]: ⚠️ Does this error go to the FE? If so we probably don't want to be this specific and have more generic error messages -- but this information would be good to log to the backend.

File: static/oas/v1/stats/penalties.yml [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        example: 1 
+      duration: 
+        type: int 
+        description: The duration the player is benched in seconds
+        example: 1
+      created_by: 
+        type: int 
+        description: The ID of the user that creates the penalty
+        example: 2
+      penalty_type_id:
```
Comment (Skylude) [inline static/oas/v1/stats/penalties.yml:118]: ⚠️ Is there a way for us to link FK tables?

**Review by Skylude — APPROVED** [review]

File: static/oas/v1/stats/penalties.yml (hunk 2 above)
Comment (ryanbarlow1) [reply static/oas/v1/stats/penalties.yml:118]: I don't believe so, but we can look deeper into this

File: backend/internal/server/apis/stats/penalty.go
Comment (ryanbarlow1) [reply backend/internal/server/apis/stats/penalty.go:1]: Thanks for this feedback, I'll think about how we want to do this

File: backend/internal/server/apis/stats/penalty.go (hunk 1 above)
Comment (ryanbarlow1) [reply backend/internal/server/apis/stats/penalty.go:27]: The error should be logged to the backend, never to the frontend. However I just tested this and it's not working as it should. I'll address that.

**Pull request merged**